contracts/DelFiPrice.sol

```diff
@@ -27,17 +32,58 @@ contract DelFiPrice is OpenOracleView {
     /// @notice The mapping of medianized prices per symbol
     mapping(string => uint64) public prices;
 
+    /// @notice Address of the cEther contract
+    address public cEthAddress;
```

Comment:
we want these as immutable 
https://solidity.readthedocs.io/en/v0.6.6/contracts.html#constant-and-immutable-state-variables